qiskit_experiments/library/mitigation/mitigation_experiment.py

```diff
@@ -0,0 +1,114 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Measurement calibration experiment classes.
+"""
+from typing import Iterable, List
+
+from qiskit import QuantumCircuit
+from qiskit.exceptions import QiskitError
+from qiskit_experiments.framework import BaseExperiment
+from .mitigation_analysis import CorrelatedMitigationAnalysis, LocalMitigationAnalysis
+
+
+class ReadoutMitigationExperiment(BaseExperiment):
+    """Class for readout mitigation experiments"""
+
+    METHOD_LOCAL = "local"
+    METHOD_CORRELATED = "correlated"
+    ALL_METHODS = [METHOD_LOCAL, METHOD_CORRELATED]
+
+    def __init__(self, qubits: Iterable[int], method=METHOD_LOCAL):
+        """Initialize a mitigation experiment.
+
+        Args:
+            qubits: The qubits being mitigated
```

Comment:
They are not being mitigated. This should be something like The qubit measurements to characterize readout error

Reply:
Done